README.md

```diff
@@ -9,18 +9,20 @@ Kurtosis: https://docs.kurtosis.com/
 
 #### Before using this repository make sure that: 
 
-- **You are using Kurtosis version 0.57.4**. This is important, as they are working on it and changes can be huge depending on different versions. You can find all Kurtosis versions [here](https://github.com/kurtosis-tech/kurtosis-cli-release-artifacts/releases).
+- **You are using Kurtosis version 0.62.0**. This is important, as they are working on it and changes can be huge depending on different versions. You can find all Kurtosis versions [here](https://github.com/kurtosis-tech/kurtosis-cli-release-artifacts/releases).
 - The topology that will be instantiated is defined in `kurtosis-module/starlark/waku_test_topology.json`. This topology is created with https://github.com/logos-co/Waku-topology-test 
 - Each node will need its own configuration file in `kurtosis-module/starlark/config_files/waku_X.toml` being `waku_X` the same name that is defined in the topology.
 
-Run this repo with:
-
-`kurtosis run main.star`
```

Comment:
Since we are forces to pass args now we should probably delete this line and make clear that the default way to start is by passing args